applications/GeoMechanicsApplication/python_scripts/geomechanics_U_Pw_solver.py

```diff
@@ -71,6 +71,7 @@ def GetDefaultParameters(cls):
             "number_cycles"              : 5,
             "increase_factor"            : 2.0,
             "reduction_factor"           : 0.5,
+            "realised_factor"            : 1.0,
```

Comment:
Perhaps this is a mistake?

Reply:
realised_factor is removed. Now corrected.